.github/workflows/pr_template_validation.yml

```diff
@@ -0,0 +1,109 @@
+name: PR Template Validation
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, ready_for_review]
+
+jobs:
+  check-pr-template:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout PR
+        uses: actions/checkout@v3
+
+      - name: Ensure Ticket Number is in Summary Section
+        run: |
+          PR_BODY=$(curl -s https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }} | jq -r '.body')
+
+          if ! echo "$PR_BODY" | grep -Pzo "(?s)^## (\[NEW-\d+\] Summary|\[NO-TICKET\] Summary|Summary)"; then
```

Comment:
I think NEW- prefix is only going to be for while we're migrating to the new ticketing system maybe replace this with `\w{3,5}` to match 3-5 word characters make it more generic / future proof

Reply:
Added up to seven characters allowed on the prefix, that would support [FEATURE-123]. I also removed case sensitivity, so if folks use [feature-123], [BUG-123], or just summary in the line they shouldn't have any issues.